test/integration/service_sign_in/create_new_account_test.rb

```diff
@@ -7,6 +7,12 @@ class CreateNewAccount < ActionDispatch::IntegrationTest
       random_example = GovukSchemas::RandomExample.new(schema: schema)
 
       payload = random_example.merge_and_validate(document_type: schema_type)
+
+      # Create new account is an optional field
+      if payload["details"]["create_new_account"].nil?
+        return
```

Comment:
Would it be better here to keep creating random examples until `create_new_account` isn't nil? Just because then it means this test is always running, rather than only running it sometimes.

Reply:
Sounds like a good idea, we'll try and add this after we ship 👍